lambda/app.py

```
@@ -910,6 +912,71 @@ def dynamic_url():
     return try_download_from_bucket(entry.bucket, entry.object_key, user_profile, custom_headers)
 
 
+@app.route('/s3credentials', methods=['GET'])
+@with_trace(context={})
+def s3credentials():
```

Should this be optional? We could either return an error response if the feature is not enabled, or not even deploy the endpoint in the cloudformation or both.

---

I think if it is going to be optional the endpoint should not be deployed.. But NASA does want temp S3 access. Is this specifically to be used with non cumulus datasets that users want direct s3 access to?

---

It should work for cumulus and NGAP as well.